rosidl_generator_cs/resource/msg.c.em

```diff
@@ -113,14 +113,14 @@ bool @(msg_typename)_native_write_field_@(member.name)(@(get_c_type(member.type.
 @[for member in message.structure.members]@
 @[  if isinstance(member.type, (AbstractSequence, Array)) and isinstance(member.type.value_type, BasicType)]@
 ROSIDL_GENERATOR_C_EXPORT
-@(get_c_type(member.type.value_type)) *@(msg_typename)_native_read_field_@(member.name)(void *message_handle, int *size)
+@(get_c_type(member.type.value_type)) *@(msg_typename)_native_read_field_@(member.name)(void *message_handle)
 {
   @(msg_typename) *ros_message = (@(msg_typename) *)message_handle;
 @[    if isinstance(member.type, Array)]@
-  *size = @(member.type.size);
+  // *size = @(member.type.size);
   return ros_message->@(member.name);
 @[    elif isinstance(member.type, AbstractSequence)]@
-  *size = ros_message->@(member.name).size;
+  // *size = ros_message->@(member.name).size;
```

Comment:
If this is well tested I would probably remove the comments

Reply:
Had not tested on Linux when I pushed this. Have done that now and the comments are sould have been removed now.